lib/pdfkit/pdfkit.rb

```diff
@@ -56,11 +56,12 @@ def executable
     PDFKit.configuration.wkhtmltopdf
   end
 
-  def to_pdf(path=nil)
+  def to_pdf(path=nil, ignore_content_errors=false)
     preprocess_html
     append_stylesheets
 
-    invoke = command(path)
+    error_log = `pwd`.chomp + '/wkhtmltopdf_errors'
```

Comment:
is it possible that this file persists and ends up with multiple errors in it? you might want to look at using `Tempfile`: https://ruby-doc.org/stdlib-1.9.3/libdoc/tempfile/rdoc/Tempfile.html

Reply:
The file is opened in wb+ mode, so it should be fine- http://ruby-doc.org/core-2.5.0/IO.html#method-c-new